JS/script.js

```diff
@@ -1,49 +1,17 @@
-const button1 = document.getElementById('6')
-
-button1.addEventListener('click', () => {
-    const cont1=document.getElementById('71')
-    cont1.style.display='block'
-});
-const button2 = document.getElementById('71')
-
-button2.addEventListener('click', () => {
-    const cont1=document.getElementById('81')
-    cont1.style.display='block'
-});
-const button3 = document.getElementById('81')
-
-button3.addEventListener('click', () => {
-    const cont1=document.getElementById('91')
-    cont1.style.display='block'
-});
-const button4 = document.getElementById('91')
-
-button4.addEventListener('click', () => {
-    const cont1=document.getElementById('101')
-    cont1.style.display='block'
-});
-let button5 = document.getElementById('101')
-
-button5.addEventListener('click', () => {
-    const cont1=document.getElementById('111')
-    cont1.style.display='block'
-});
-let button6 = document.getElementById('12')
 
+for (let i=5;i<14;i++) {

```

Comment:
what if I decide to delete element with id "7"? The script will fail

Reply:
my bad, have no clue how to do this in other ways